### PR TITLE
[HPL-717]: test v3 codes

### PR DESCRIPTION
**Pull request by hashableric**
- testing beneath contracts
  - default-hook
  - domain-routing-hook

### CODE REVIEW

File: contracts/default-hook/src/bin/schema.rs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+use std::{env::current_dir, fs::remove_dir_all};
+
+use cosmwasm_schema::write_api;
+
+use hpl_interface::domain_routing_hook::{ExecuteMsg, InstantiateMsg, MigrateMsg, QueryMsg};
```
Comment (byeongsu-hong) [inline contracts/default-hook/src/bin/schema.rs]: domain_hook

File: contracts/default-hook/src/query.rs [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+    #[test]
+    fn test_quote_dispatch() {
+        let mut deps = mock_dependencies();
+
+        deps.querier.update_wasm(
+            |_| -> cosmwasm_std::SystemResult<cosmwasm_std::ContractResult<cosmwasm_std::Binary>> {
+                let res = cosmwasm_std::ContractResult::Ok(
+                    to_binary(&QuoteGasPaymentResponse {
+                        gas_needed: Uint256::from_str("1000000").unwrap(),
```
Comment (byeongsu-hong) [inline contracts/default-hook/src/query.rs:111]: :+1:

File: contracts/default-hook/src/state.rs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+pub const HOOK_CONFIG_KEY: &str = "hook_config";
+pub const HOOK_CONFIG: Map<u32, HookConfig> = Map::new(HOOK_CONFIG_KEY);
+
+pub const CUSTOM_HOOK_CONFIG_KEY: &str = "custom_hook_config";
+pub const CUSTOM_HOOK_CONFIG: Map<Vec<u8>, HookConfig> = Map::new(CUSTOM_HOOK_CONFIG_KEY);
+
+pub const PAUSE_KEY: &str = "pause";
+pub const PAUSE: Item<bool> = Item::new(PAUSE_KEY);
+
+pub fn generate_hook_key(destination: u32, recipient: Binary) -> Vec<u8> {
```
Comment (byeongsu-hong) [inline contracts/default-hook/src/state.rs]: why don't you use this in testcode?

**Review by byeongsu-hong — CHANGES_REQUESTED** [review]

File: contracts/default-hook/src/bin/schema.rs (hunk 1 above)
Comment (hashableric) [reply contracts/default-hook/src/bin/schema.rs]: @byeongsu-hong same naming with ethereum contract.

File: contracts/default-hook/src/state.rs (hunk 3 above)
Comment (hashableric) [reply contracts/default-hook/src/state.rs]: @byeongsu-hong i donno what you blame for

**Review by byeongsu-hong — APPROVED** [review]

**Pull request merged**